llvm/include/llvm/IR/FPAccuracy.def

```diff
@@ -0,0 +1,66 @@
+//===--- llvm/IR/FPAccuracy.def - Mappings for fp accuracy  -----*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Defines properties of floating point builtin intrinsics.
+//
+//===----------------------------------------------------------------------===//
+
+// This customization is only in xmain for review purposes.
+// When approved, it will go into the intel/llvm repository.
+
+#ifndef FP_ACCURACY
+#define FP_ACCURACY(IID,SF,SD,CF,CD
+#endif
+
+// Each entry below maps an fpbuiltin intrinsic ID to the required accuracy
+// for that operation for single- and double-precision for SYCL and CUDA
+//
+// All accuracies are returned as single-precision floating-point values.
+//
+// Note: for single-precision fdiv and sqrt, the value returned here assumes
+// that the "-cl-fp32-correctly-rounded-divide-sqrt" option is not used.
+// If the option is used, these operations require special handling elsewhere
+//
+// FP_ACCURACY(<IID>, <SYCL_FLOAT>, <SYCL_DOUBLE>, <CUDA_FLOAT>, <CUDA_DOUBLE>)
+//
+FP_ACCURACY(fpbuiltin_fadd,   "0.0f",  "0.0f",  "0.0f",  "0.0f")
```

Comment:
I wasn't expecting the 'f' on the end of the string, but if APFloat parses that correctly it's probably better.